.travis.yml

```diff
@@ -3,7 +3,13 @@ sudo: required
 language: node_js
 node_js: 10
 
+env:
+  - PATH=$HOME/purescript:$PATH
 install:
+  - TAG=$(wget -q -O - https://github.com/purescript/purescript/releases/latest --server-response --max-redirect 0 2>&1 | sed -n -e 's/.*Location:.*tag\///p')
+  - wget -O $HOME/purescript.tar.gz https://github.com/purescript/purescript/releases/download/$TAG/linux64.tar.gz
+  - tar -xvf $HOME/purescript.tar.gz -C $HOME/
+  - chmod a+x $HOME/purescript
```

Comment:
Is this necessary now that 0.12 is on npm?

Reply:
I guess not, but it's hell of a lot faster to install

Reply:
I'm more worried about my mental capacity. :wink:
I'll probably leave it in if no one else cares.

Reply:
Personally, I prefer the current version since it pins the compiler version.